.github/workflows/ubuntu-workflow.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-18.04
```

Comment:
then we could have issues with performance tests

Reply:
@piotradamczyk5 will test again a few more things

Reply:
anyway this blocks us, so it is better to merge and once again fix the performance test issue

Reply:
tried with latest too

Reply:
latest for now is 
```
Ubuntu
  18.04.5
  LTS
```
but it will be changed to 20.04 soon

Reply:
The latest build passes on the current configuration https://github.com/Flank/flank/pull/1521/checks?check_run_id=1737489139
Maybe that was just a temporary issue?